Coordinator/Coordinator/View/MainViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MainViewController.swift
+//  Coordinator
+//
+//  Created by Vladka on 28.12.2022.
+//
+
+import UIKit
+
+/// Главная страница приложения
+final class MainViewController: UIViewController, Storyboarded {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let isKeyText = "isLogin"
+        static let mapText = "Map"
+    }
+
+    // MARK: - Public Properties
+    
+    var onLogout: (() -> Void)?
+    
+    // MARK: - IBAction
+    @IBAction private func logout(_ sender: Any) {
```

Comment:
```suggestion
    @IBAction private func logoutAction(_ sender: Any) {
```
везде не буду писать, но возможно стоит добавить Action во всех кнопках